common/components/BalanceSidebar/AccountInfo.tsx

```diff
@@ -69,6 +72,14 @@ class AccountInfo extends React.Component<Props, State> {
     });
   };
 
+  public onCopy = () => {
+    this.setState(state => {
+      return {
+        copied: !state.copied
+      };
+    });
+  };
```

Comment:
Right now the only way to get rid of the copied text is to click it again. That also doesn't give exactly the right impression, since they just copied it again. I'd suggest leaving this text up just for a little bit, something like:

```
this.setState({ copied: true });

setTimeout(() => {
  this.setState({ copied: false });
}, 2000);
```

Reply:
I also realize that you can't really undo a copy to clipboard. I think I'll just remove that aspect, and keep the copied text displayed